.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
```

Comment:
@v-i-s-h-n-u-ps I have some opinions about this 😅 

Do you feel strongly about making this pre-commit or would you be okay moving this to a pre-push? IMHO, pre-commit hooks disrupt the developer's workflow and do more harm than good. I also don't love the fix tag, I think the automatic run should show you the error but shouldn't fix by itself. 

I'm happy to have a conversation about his if you feel strongly. My proposal is to make this a pre-push hook and remove the `--fix` (this part is less important). Here is an example from one of our other repos: https://github.com/Constructor-io/constructorio-client-javascript/blob/master/.husky/pre-push